backend/src/ai/_ollama.py

```
@@ -0,0 +1,79 @@
+from typing import Dict
+from .classes import ConversationManager
+from ollama import AsyncClient
+from ..config import OLLAMA_HOST, OLLAMA_HEADERS # The work directory will ALWAYS be backend/
+from ..database import get_sessions
+from bson import ObjectId
+
+try:
+  if OLLAMA_HOST is None:
+    raise ValueError("Ollama host is not set in config.ini")
+except KeyError:
+  raise ValueError("Ollama host is not set in config.ini")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)